games-board/cockatrice/cockatrice-20210126.ebuild

```
@@ -3,14 +3,14 @@
 
 EAPI=7
 
-inherit cmake
+inherit cmake xdg-utils
```

```suggestion
inherit xdg cmake
```
and then kill the `pkg_*` phases